lib/testy.js

```diff
@@ -178,6 +178,7 @@ class Testy {
 
   async #loadFileHandlingErrors(file) {
     try {
+      testRunner.loadingFile(file);
```

Comment:
I'd like to explore how difficult it'll be to store/display relative paths, not absolute. But if it's too much effort/complexity to the code, I am ok with having absolute paths